src/core/tests/frontend/paddle/test_models/gen_scripts/generate_elementwise_ops.py

```diff
@@ -162,6 +162,32 @@ def elementwise_pow(name : str, x, y, axis, in_dtype):
 
     return outs[0]
 
+
+def elementwise_floordiv(name : str, x, y, axis, in_dtype, paddle_ver="1.8"):
+    import paddle
+    paddle.enable_static()
+
+    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
+        node_x = paddle.static.data(name = 'x', shape = x.shape, dtype = in_dtype)
+        node_y = paddle.static.data(name = 'y', shape = y.shape, dtype = in_dtype)
+        if paddle_ver == "1.8":
```

Comment:
We have no intention to support paddle 1.8. So I would suggest to remove corresponding code and test cases.

Reply:
Think twice. Let's settle down this version after we know more about "axis" discrepancy.

Reply:
this is only for test different api in paddle;  other elementwise_op only support paddle 1.8; when those elementwise ops support paddle2.x ,  you want to remove corresponding code and test cases .

Reply:
Can you please use paddle.__version__ to detect paddle version, instead of using the argument paddle_ver?

Reply:
> if use paddle.**version**, axis != -1 cannot add in tese cases, you want test axis == -1 ?

We have no intention to support paddle 1.8. We don't hope the name of this argument bring any confusion to developers and users. 
But in the op mapper, we could keep your implementation to make some legacy paddle models are supported as well.

So can you simply create the test cases without the implication of paddle version?

Reply:
Probably simply use both fluid and paddle 2.1 api to create cases. That's all.

Reply:
update